src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework/QueueItem.cs

```diff
@@ -11,11 +11,11 @@
 namespace Microsoft.CommonLanguageServerProtocol.Framework;
 
 /// <summary>
-/// A placeholder type to help handle Notification messages.
+/// A placeholder type to help handle parameterless messages and messages with no return value.
 /// </summary>
-internal record VoidReturn
+internal sealed class NoValue
 {
-    public static VoidReturn Instance = new();
+    public static NoValue Instance = new();
```

Comment:
I couldn't find anyone else using this, so this might not need a dual insertion

Reply:
It's internal, so shouldn't be used, no?

Reply:
I believe everyone using this has IVTs